### DISCUSSION TITLE
Marking datasets for models.

### DISCUSSION THREAD

**Discussion by bropines** [civitai/civitai#495] [Ideas]
It is possible to make a dataset markup service based on civitai. 
That is, conventionally, anyone registering can specify that this picture of a girl, a cat and a cup. 
It will also be possible to upload your own pictures to datasets, matching the style / theme.

**Comment by JustMaier** [comment]
This is something that's been on my mind a lot lately. I really like the idea of being able to open up the ability for the community to add data to images. I'll need to discuss this with the team to see the right way to handle this.

**Reply by bropines** [reply]
> This is something that's been on my mind a lot lately. I really like the idea of being able to open up the ability for the community to add data to images. I'll need to discuss this with the team to see the right way to handle this.

Then, if you decide to start, then my friend and I are ready to help with the backend / frontend in which case. It was just boring for us to mark up 1k photos for half an evening and this idea surfaced. My telegram, in which case @bropines
